access.go

```diff
@@ -91,6 +91,9 @@ type AccessData struct {
 
 	// Data to be passed to storage. Not used by the library.
 	UserData interface{}
+
+	// Extend token expiration
+	ExtendExpiryDate bool
```

Comment:
Extend to what?

Reply:
this for enable remember me feature. if true, token expiration will be extended to a certain duration set in environment variable